src/workerd/api/url-standard.c++

```diff
@@ -53,6 +53,17 @@ bool URL::canParse(kj::String url, jsg::Optional<kj::String> maybeBase) {
   return jsg::Url::canParse(url, maybeBase.map([](kj::String& str) { return str.asPtr(); }));
 }
 
+jsg::JsString URL::createObjectURL(
+    jsg::Lock& js, kj::OneOf<jsg::Ref<File>, jsg::Ref<Blob>> object) {
+  // TODO(soon): Implement this
```

Comment:
Might be worth expanding a bit on what needs to be considered to implement this. It's not trivial by any stretch.

The `createObjectURL(...)` method is expected to store the `Blob` or `File` persistently in some scope such that the returned blob URL can be used to fetch it from that scope. In browsers and Node.js this is relatively straightforward. In workers, however... would that scope be expected to be single-metal? colo-wide? fleet-wide? If the URL is passed to a fetch on a different metal would it be expected to work? Where is the data actually stored? Is it volative in memory such that these URLs are also fully volatile? If they are fully volatile is there a risk of collisions? As so on... Implementing these is not trivial and we cannot simply copy what Node.js has done.

Reply:
Agreed. I'll be creating an internal proposal to see what our options are.